src/utils/roles.js

```diff
@@ -0,0 +1,22 @@
+// ! HELPER FUNCTIONALITIES FOR DATA PERTAINING TO USER ROLES
+
+// * PKGS
+import axios from 'axios';
+
+// * RETURN THE ROLE OF THE LOGGED IN USER AND SET STATE DATA
+export const validateRole = async (data, set) => axios.post(
+    'http://localhost:8888/api/users/',
```

Comment:
should this still be local host?

Reply:
Need clarity from Bao on this, will resolve asap.